docs_user/modules/proc_migrating-tls-everywhere.adoc

```diff
@@ -2,29 +2,24 @@
 
-If the {OpenStackPreviousInstaller} deployment was deployed with TLS-e, FreeIPA (IdM) is used to issue certificates for the {OpenStackShort} services. Certmonger, a client process which is installed on all hosts, interacts with FreeIPA (IdM) to request, install, track and renew these certificates.
+The {rhos_acro} deployment uses the cert-manager operator to issue, track, and renew the certificates. In the following procedure, you extract the CA signing certificate from the FreeIPA instance that you use to provide the certificates in the {OpenStackShort} environment, and then import them into cert-manager in the {rhos_acro} environment. As a result, you minimize the disruption on the Compute nodes because you do not need to install a new chain of trust.
 
-The {rhos_acro} Operator-based deployment uses the cert-manager operator to issue, track, and renew the certificates.
+You then decommission the previous FreeIPA node and no longer use it to issue certificates. This might not be possible if you use the IPA server to issue certificates for non-{OpenStackShort} systems.
 
-Because the same root certificate authority (CA) is used to generate new certificates, you do not have to modify the currently used chain of trust.
-_Disclaimer: the below steps were reproduced on a FreeIPA 4.10.1 server. The location of files and directories may slightly change on different versions._
-
-These instructions explain how to extract the CA signing certificate from the FreeIPA instance that is used to provide the certificates in the source environment and import it into certmanager for use in the target environment. In this way, disruption on the Compute nodes can be minimized because a new chain of trust need not be installed.
-
-It is expected that the old FreeIPA node is then decommissioned and no longer used to issue certificates. This might not be possible if the IPA server is used to issue certificates for non-{OpenStackShort} systems.
+[NOTE]
+====
+* The following procedure was reproduced on a FreeIPA 4.10.1 server. The location of the files and directories might change depending on the version.
 
-This procedure will also need to be modified if the signing keys are stored in an hardware security module (HSM) instead of an NSS shared database (NSSDB). In that case, if the key is retrievable, special HSM utilities might be required.
-//kgilliga: Note to self: This intro will need to be rewritten/reorganized. Determine the most relevant info.
+* If the signing keys are stored in an hardware security module (HSM) instead of an NSS shared database (NSSDB), and the keys are retrievable, special HSM utilities might be required.
+====
 
 .Prerequisites
 
 * Your {OpenStackShort} deployment is using TLS-e.
-* Make sure the previous Adoption steps (if any) have been performed successfully.
-* Make sure the backend services on the new deployment are not started yet.
```

Comment:
I removed the "Make sure the previous adoption steps..." prerequisite because there isn't much that comes before this prcoedure  besides reading the planning information and doing the “Comparing configuration files between deployments” procedure. Do we want to call out that procedure specifically?

Reply:
nope - thats fine I think.